src/commands/backup.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Command, flags } from '@oclif/command';
+import { BootstrapService, BootstrapUtils } from '../service';
+
+export default class Backup extends Command {
+    static description = `The command backups the Mongo and RocksDb data folder into a Zip file that can be used for --backupSync feature. Bootstrap compose services must be stopped before calling this command.
```

Comment:
```suggestion
    static description = `The command backs up the Mongo and RocksDb data folder into a Zip file that can then be used by the --backupSync feature. Bootstrap compose services must be stopped before calling this command.
```

How can we surround `--backupSync` with backticks so that it appears in monospace in all generated docs?